src/pages/ComicBreakdownPage.tsx

```diff
@@ -1,26 +1,39 @@
 import { Divider, Button } from "@nextui-org/react";
-import { IComicData } from "../models/comicData";
 import Navigation from "../elements/Navigation";
+import { useQuery } from "@tanstack/react-query";
+import { getComicInfo } from "../utilities/api";
+import { getComicBreakdown } from "../utilities/transformAPIdata";
 
-function ComicBreakdownPage({ comic }: { comic: IComicData }) {
-  return (
-    <>
-      <Navigation isLoggedIn={true}></Navigation>
-      <div className="max-w-md">
-        <div className="space-y-1">
-          <h4 className="text-medium font-medium">{comic.name}</h4>
-        </div>
-        <Divider className="my-4" />
-        <div className="flex h-5 items-center space-x-4 text-small">
-          <div>Image</div>
-          <Divider orientation="vertical" />
-          <div>Comic 'Meta' data</div>
+function ComicBreakdownPage({ slug }: { slug: string }) {
+  const { isPending, isLoading, isError, data } = useQuery({
+    queryKey: ["search", slug],
```

Comment:
Nice use of the slug within the queryKey list to make sure the query is unique to that comic response 👏